frame/state-trie-migration/src/lib.rs

```diff
@@ -617,7 +621,7 @@ pub mod pallet {
 				let (_imbalance, _remainder) = T::Currency::slash(&who, deposit);
 				Self::deposit_event(Event::<T>::Slashed { who, amount: deposit });
 				debug_assert!(_remainder.is_zero());
-				return Err(Error::<T>::SizeUpperBoundExceeded.into())
```

Comment:
Really small nit pick, but it might make sense to emit this error "SizeUpperBoundExceeded" as an event, unless this is the ONLY way that the `Slashed` event can be emitted, then it would be the same

Reply:
Yeah we only slash for the same reason everywhere.